modules/internal/Atomics.chpl

```diff
@@ -281,6 +323,10 @@ module Atomics {
        Similar to :proc:`compareExchange`, except that this function may
        return `false` even if the original value was equal to `expected`. This
        may happen if the value could not be updated atomically.
+
+       This weak version is allowed to spuriously fail, but when
+       is used compareExchange in a loop anyways, it can can offer better
```

Comment:
I think there's a missing word between "when" and "is"?  Also "can" turns up twice in a row

Reply:
It looks like it got copied to other places as well, so those will also need an update